content/en/cloud_cost_management/planning/commitment_programs.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Commitment Programs
+description: Learn how to manage the performance and status of your cloud discount programs.
+further_reading:
+- link: "/cloud_cost_management/"
+  tag: "Documentation"
+  text: "Learn about Cloud Cost Management"
+---
+
+<div class="alert alert-info">CCM Commitment Programs supports Amazon RDS Reserved Instances and Amazon EC2 Reserved Instances.</div>
+
+## Overview
+
+Cloud providers offer commitment-based discount programs, such as {{< tooltip text="Reserved Instance (RI)" tooltip="A billing discount for committing to use a specific instance configuration for a one- or three-year term." >}} and {{< tooltip text="Savings Plans" tooltip="Flexible cloud discount programs that provide lower prices in exchange for a commitment to a consistent amount of usage (measured in $/hour) over a term." >}}, to help you save on predictable usage. Datadog's Commitment Programs feature helps you monitor, optimize, and maximize the value of these discounts across your cloud environments.
+
+With Commitment Programs, you can:
+- Track and address unused or underused commitments
+- Target high {{< tooltip text="on-demand" tooltip="Cloud resources billed at standard rates, without any commitment or discount program." >}} spend with additional commitments
+- Monitor expirations and plan timely renewals
+
+{{< img src="cloud_cost/planning/planning-commitments-overview.png" alt="Dashboard showing filters, KPIs (Effective Savings Rate, Absolute Savings, Coverage, Utilization), a cost-over-time bar chart, and a table of on-demand hot-spots by region, instance family, and database engine." style="width:100%;" >}}
+
+This page explains how to use the [Commitment Programs page][1] to understand and optimize your cloud commitments.
+
+## Getting started
+
+1. Go to [**Cloud Cost > Planning > Commitment Programs**][1] in Cloud Cost Management.
+2. Review KPIs in the [Commitments Overview](#commitments-overview) section.
+3. Use filters to focus on specific accounts, regions, or services.
+4. Explore the [Costs Overview](#costs-overview) to analyze utilization, coverage, and expiring commitments.
```

Comment:
I might have missed this, but where is the info on expiring commitments?

Reply:
removed that mention from Costs Overview - it is only in Commitments Overview and Commitments Explorer.